spec/models/service_template_transformation_plan_request_spec.rb

```diff
@@ -1,10 +1,13 @@
+require 'byebug'
```

Comment:
How'd that sneak in ;)

Reply:
Don't know. Where have you seen it ? :laughing: